src/main/java/com/unzer/payment/service/Configuration.java

```diff
@@ -0,0 +1,91 @@
+package com.unzer.payment.service;
+
+/*-
+ * #%L
+ * Unzer Java SDK
+ * %%
+ * Copyright (C) 2020 - today Unzer E-Com GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.unzer.payment.exceptions.PropertiesException;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class Configuration {
+    private static final String API_ENVIRONMENT = "unzer.environment";
+    private static final String PRODUCTION_ENDPOINT = "https://api.unzer.com";
```

Comment:
suggestion: here is a chance to split up the URL in different parts to just have on base url eg. "https//{envPrefix}{domain}"